### PR TITLE
[OYPD-338] Add permission 'Access admin structure page' and replace the structure page route permission control.

### PR DESCRIPTION
**Pull request by Kerby70**
Jira https://propeople-us.atlassian.net/browse/OYPD-338

Steps to review
- [x] Login as Editor role
- [x] See menu has Structure link
- [x] Sub menu items under structure should contain Menus and Taxonomy

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build742 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build742 |
| Installation process | http://install.openy-dev.ffwua.com/build742/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/742/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build742/profiles/openy/build/reports/behat

File: modules/openy_features/openy_user/config/install/user.role.editor.yml [hunk 1]
```diff
@@ -145,3 +145,4 @@ permissions:
   160: 'edit terms in media_tags'
   161: 'administer menu'
   162: 'access tour'
+  163: 'access structure page'
```
Comment (alexschedrov) [inline modules/openy_features/openy_user/config/install/user.role.editor.yml:148]: Upgrade path.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/743/

**Build Log**
*last 25 lines*

 ```
[...truncated 225 lines...]
stdout: You are about to create a /var/www/build743/sites/default/settings.php file and DROP all tables in your 'drupal743' database. Do you want to continue? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=14   changed=6    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by AlexNetman** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/748/

**Build Log**
*last 25 lines*

 ```
[...truncated 225 lines...]
stdout: You are about to create a /var/www/build748/sites/default/settings.php file and DROP all tables in your 'drupal748' database. Do you want to continue? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=14   changed=6    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/749/

**Build Log**
*last 25 lines*

 ```
[...truncated 233 lines...]
stdout: You are about to create a /var/www/build749/sites/default/settings.php file and DROP all tables in your 'drupal749' database. Do you want to continue? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=14   changed=6    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build750 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build750 |
| Installation process | http://install.openy-dev.ffwua.com/build750/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/750/

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build752 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build752 |
| Installation process | http://install.openy-dev.ffwua.com/build752/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/752/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build752/profiles/openy/build/reports/behat

**Comment by alexschedrov** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build755 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build755 |
| Installation process | http://install.openy-dev.ffwua.com/build755/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/755/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build755/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**